src/components/DocumentationTopic/Hero/Hierarchy.vue

```diff
@@ -134,11 +134,7 @@ export default {
         return all;
       }, []);
     },
-    /**
-     * Extract the root item from the parentTopics.
-     * Works only if above 1000px windowWidth
-     */
-    root: ({ parentTopics, windowWidth }) => (windowWidth <= 1000 ? null : parentTopics[0]),
+    root: ({ parentTopics }) => parentTopics[0],
```

Comment:
Good catch but we want to always display a root since we deleted the "Documentation" text that was coming before. I transformed the firstItemSlice into a const instead of a computed property. https://github.com/apple/swift-docc-render/pull/772/commits/88538f340c3a7435af62e2ec0d4f7861cb35a94c